hooks/validators/deny_unsafe_capabilities.go

```diff
@@ -0,0 +1,60 @@
+package validators
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// default list of capabilities for Docker
+// ref: https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities
+var defaultCapabilities = []string{
+	"AUDIT_WRITE",
+	"CHOWN",
+	"DAC_OVERRIDE",
+	"FOWNER",
+	"FSETID",
+	"KILL",
+	"MKNOD",
+	"NET_BIND_SERVICE",
+	"NET_RAW",
+	"SETFCAP",
+	"SETGID",
+	"SETPCAP",
+	"SETUID",
+	"SYS_CHROOT",
+}
+
+// DenyUnsafeCapabilities is a Validator that denies adding capabilities beyond the default set
+func DenyUnsafeCapabilities(ctx context.Context, pod *corev1.Pod) field.ErrorList {
```

Comment:
The capabilities field in PSP accepts a list of capabilities to be allowed.
I'm not sure this is needed right now, but please consider it is needed or not.